README.md

```diff
@@ -1 +1,75 @@
-# First-Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone project
+
+
+> This is a project for a climate change summit website. It contains details about the conference such as the speakers, the date, venue and the main speakers.
+
+
+## Built With
+
+- Major languages:
+  - HTML
+  - CSS
+  - Javascript
+- Frameworks
+  - Bootstrap 5
+- Technologies
+  - Linters
+
+## Demo Links
+
+- [Live Demo Link]([https://aquinasmoriasi.github.io/First-Capstone-Project/](https://drive.google.com/file/d/1IYAXnYJ6rdBYw5BQzioRgXQ8uvp8CC6T/view?usp=sharing))
```

Comment:
- [ ] I think you made a mistake by adding the video presentation link instead of the live demo link. Please update the Live demo link with the proper one.
- [ ] Try to add another link to the video presentation to your Readme or your pull request description. The link should be like this:
```
[Video](https://drive.google.com/file/d/1IYAXnYJ6rdBYw5BQzioRgXQ8uvp8CC6T/view?usp=sharing)
```

Reply:
I see my bad.